client/src/components/layout/HelForm.jsx

```diff
@@ -102,13 +102,14 @@ const HelForm = ({ submitForm, submitted, user }) => {
   };
 
   const handleCourseAddition = e => {
-    if (!humanitiesCourses.includes(e.target.innerHTML)) {
+    let event = (e.target.innerHTML).toLowerCase();
+    if (!humanitiesCourses.includes()) {
```

Comment:
```
!humanitiesCourses.includes()
```
 seems to be a redundant check. Did you have something in mind when you changed this?